homeworkConcurrency/src/main/java/homework07/Festival.java

```diff
@@ -0,0 +1,30 @@
+package homework07;
+
+import java.util.LinkedList;
+import java.util.Random;
+
+public class Festival {
+
+    TicketType ticketType;
+
+    public void data() {
+
+        LinkedList<TicketType> list = new LinkedList();
+        FestivalGate gate = new FestivalGate("enterace", 1);
+
+        TicketType ticketType = generateRandomData();
+
+        FestivalStatisticsThread statsThread = new FestivalStatisticsThread(list);
+        statsThread.start();
+
+        FestivalAttendeeThread festivalAttendeeThread = new FestivalAttendeeThread(gate, ticketType, list);
```

Comment:
was expecting a for with multiple threads, not a thread with a for, but still works (you could fix for future references)